en/components/general/localization.md

```diff
@@ -1,32 +1,87 @@
 ---
-title: Localization - Native Angular | Ignite UI for Angular
+title: Localization (i18n) - Native Angular | Ignite UI for Angular
 _description: The Ignite UI for Angular Localization mechanism provides the ability to change/localize strings in the components.
 _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Native Angular Components
 ---
 
-# Localization
+# Localization (i18n)
+
+Currently, Ignite UI for Angular ships with resource strings for the following languages: German, French, Italian, Japanese, Korean, Spanish, Traditional Chinese (zh-Hant) and Simplified Chinese (zh-Hans). These are available via the `igniteui-angular-i18n` package.
```

Comment:
...for the following languages **and scripts**
...**traditional and simplified Chinese** (I think there is no need for the zh-Hant and zh-Hans).